celery/backends/couchbase.py

```diff
@@ -39,13 +39,16 @@ class CouchbaseBackend(KeyValueStoreBackend):
     username = None
     password = None
     quiet = False
+    supports_autoexpire = True
+
     timeout = 2.5
 
     # Use str as couchbase key not bytes
     key_t = str_t
 
     def __init__(self, url=None, *args, **kwargs):
-        super(CouchbaseBackend, self).__init__(*args, **kwargs)
+        super(CouchbaseBackend, self).__init__(expires_type=int, *args,
```

Comment:
Perhaps a better form would be to add this option in the keyworded arguments if it does not exist:

```
kwargs.setdefault('expires_type', int)
```